ONSdigital/response-operations-ui
# Survey page overview

# Motivation and Context
ability to add survey mode when creating survey

# What has changed
added funcitonality

# How to test?
sign in to rops, create a survey, click on the survey. It should say eQ or SEFT in the Survey Mode section.
